package.json

```diff
@@ -60,6 +60,7 @@
 		"./data-type": "./dist-cms/packages/settings/data-types/index.js",
 		"./language": "./dist-cms/packages/settings/languages/index.js",
 		"./logviewer": "./dist-cms/packages/settings/logviewer/index.js",
+		"./webhooks": "./dist-cms/packages/settings/webhooks/index.js",
```

Comment:
@iOvergaard do we want singular / plural form here? e.g. tag/tag, language/languages and webhook/webhooks?

How about repository name? Would be great to document some name conventions to follow.

Reply:
Singular form, please 👍 

I would also really like it if you set it up as a new "package" inside the packages folder. It could be called "headless" or something like that. Then it will consist of a Webhook workspace, but we will also be able to add the "Preview URLs" extension to that package, and whatever functionality comes in the future.

The "Settings"-package will go away at some point. I just need to find the time to move the other areas to some good places 😃 

The repository name will be singular too. `UmbWebhookRepository`. The repository might also make sense to split into multiple smaller repositories, but I am not 100% sure what functionality is part of webhooks so you can just leave it in one, if it doesn't make sense how to split it.

Let me know if you run into trouble.

Reply:
I guess we can move it here as a "webhooks" package:
https://github.com/umbraco/Umbraco.CMS.Backoffice/tree/main/src/packages

Not sure "headless" makes sense as I guess traditional projects could benefits webhooks as well without being headless and since it is added to CMS project in Umbraco 13.